content/influxdb/v2.0/reference/cli/influxd/upgrade/_index.md

```diff
@@ -0,0 +1,36 @@
+---
+title: influxd upgrade
+description: The `influxd upgrade` command upgrades a InfluxdDB 1.x instance to 2.0.
+menu:
+  influxdb_2_0_ref:
+    parent: influxd
+weight: 201
+products: [oss]
+---
+
+Use the `influxd upgrade` command to upgrade an instance of InfluxDB 1.x to InfluxDB 2.0.
+This will copy all data in [databases](/influxdb/v1.8/concepts/glossary/#database) and 
+[retention policies](/influxdb/v1.8/concepts/glossary/#retention-policy-rp)(used in 1.x)
+over to [buckets](/influxdb/v2.0/reference/glossary/#bucket) (used in 2.0).
+
+## Usage
```

Comment:
I don't know where to put this, but somewhere we need a big warning that says they should backup their data before upgrading.